src/helpers/getItemsFromCart.js

```diff
@@ -0,0 +1,10 @@
+const { expect } = require("@playwright/test");
+
+export async function getItemsFromCart(shopingCartPage){
```

Comment:
This method should be a part of `shopingCartPage`

Reply:
not sure if this ok because I re-use this in checkout.spec.js
In general, I think I can include this func in helpers.js file